data/charts/crds/templates/crds.yaml

```diff
@@ -1,3 +1,12 @@
+# To prevent accidental injection into istio-system.
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: {{ .Release.Namespace }}
+  labels:
+    istio-operator-managed: Reconcile
+    istio-injection: disabled
+---
```

Comment:
We should start namespaceing new labels, e.g. 

```yaml
operator.istio.io/managed: Reconcile
```

We'll need to migrate existing labels over time. xref https://github.com/istio/istio/issues/14777

Reply:
Done.